api/tests/acceptance/application/session-controller_test.js

```diff
@@ -55,9 +57,6 @@ describe('Acceptance | Controller | session-controller', function() {
       }
     };
 
-    before(() => knex('sessions').delete());
```

Comment:
C'était bien ça, on s'assurait que tout soit propre avant de commencer les tests :)

Reply:
Not needed we have an after each who delete at start